content/blog/translating-rmd.md

```diff
@@ -0,0 +1,132 @@
+---
+author: "Rob Ashton"
+date: 2021-06-30
+title: Translating R Markdown
+best: false
+tags:
+ - R
+ - rmarkdown
+ - translation
+---
+
+## Translating Rmd
+
+As part of our work on [naomi](../../projects/naomi) we've been asked to produce reports from R which can be distributed to country teams across Sub-Saharan Africa in English, French and Portuguese. R Markdown has no prescribed method for translating documents, but there are many options available to make this possible. This post covers 3 possible approaches: code chunks, custom blocks and tabsets.
```

Comment:
Just in case anyone is familiar with R but not R Markdown then you could add a link:
```suggestion
As part of our work on [naomi](../../projects/naomi) we've been asked to produce reports from R which can be distributed to country teams across Sub-Saharan Africa in English, French and Portuguese. [R Markdown](https://rmarkdown.rstudio.com/docs/) has no prescribed method for translating documents, but there are many options available to make this possible. This post covers 3 possible approaches: code chunks, custom blocks and tabsets.
```